lavender.json

```diff
@@ -1,28 +1,28 @@
 {
 	"response": [
 		{
-			"maintainer": "",
+			"maintainer": "Vhmit",
 			"oem": "Redmi",
 			"device": "Redmi Note 7",
-			"filename": "crDroidAndroid-13.0-20230813-lavender-v9.8.zip",
-			"download": "https://sourceforge.net/projects/crdroid/files/lavender/9.x/crDroidAndroid-13.0-20230813-lavender-v9.8.zip/download",
-			"timestamp": 1691937971,
-			"md5": "eaca7ab5696b4f4fe11ae40298c543a2",
-			"sha256": "9822eccec7b28e8285e073dce6ab1b57ae7310ce359a3d5941142648dcfbbcf5",
-			"size": 1228958706,
-			"version": "9.8",
+			"filename": "crDroidAndroid-13.0-20231028-lavender-v9.10.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/lavender/9.x/crDroidAndroid-13.0-20231028-lavender-v9.10.zip/download",
+			"timestamp": 1698459883,
+			"md5": "f982219ed474db96a13c83c42b48acbc",
+			"sha256": "4e0a066fa0045ed254543c9a7e2408896118d2b300b14549f35bdf79316b07a9",
+			"size": 1062910259,
+			"version": "9.10",
 			"buildtype": "Monthly",
-			"forum": "https://forum.xda-developers.com/t/rom-13-0-official-lavender-crdroid-v9-8-14-08-2023.4523341/",
+			"forum": "https://xdaforums.com/t/rom-13-0-official-lavender-crdroid-v9-10-28-10-2023.4638733/",
 			"gapps": "https://sourceforge.net/projects/nikgapps/files/Releases/",
 			"firmware": "",
 			"modem": "",
 			"bootloader": "",
-			"recovery": "https://sourceforge.net/projects/lc-dev/files/lavender/TWRP-recovery-erofs-dynamic-partitions-230713.img/download",
-			"paypal": "https://www.paypal.me/prashant1605",
-			"telegram": "https://t.me/NexLav",
-			"dt": "https://github.com/crdroidandroid/android_device_xiaomi_lavender",
+			"recovery": "https://sourceforge.net/projects/heimerdinger-files/files/Recoverys/lavender/A13-OrangeFox-lavender-stable%40R11.1_5.zip/download",
+			"paypal": "",
+			"telegram": "https://t.me/Vhmit",
+			"dt": "https://github.com/crdroidandroid/device_xiaomi_lavender",
```

Comment:
I've renamed this one to keep standard, so please update
android_device_xiaomi_lavender